english/eating-walls/eating-marshmallows.md

```diff
@@ -6,8 +6,7 @@ this story happened to provide you. Shoveling the fluffy, whipped sugar into you
 you begin to feel a sense of satisfaction.
 
 Sitting down next to the partially eaten marshmallow wall, your stomach begins to grumble, churn,
-and babble. Hoping to walk off the uneasy feeling, you stand up. Upon standing up, your stomach
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.